quarkus/service/src/main/resources/application.properties

```diff
@@ -119,29 +119,5 @@ polaris.authentication.token-broker.max-token-generation=PT1H
 # polaris.storage.gcp.lifespan=PT1H
 
 %test.quarkus.log.file.enable=false
-%test.quarkus.log.category."org.apache.polaris".level=INFO
-%test.quarkus.log.category."org.apache.iceberg.rest".level=INFO
-%test.quarkus.log.category."org.apache.iceberg.rest.RESTSessionCatalog".level=ERROR
```

Comment:
Removing these loggers for now, let's see if users complain about logs being too verbose.